src/gar.lua

```diff
@@ -2233,4 +2233,37 @@ function gar.getPaginatedDelegatesFromAllGateways(cursor, limit, sortBy, sortOrd
 	)
 end
 
+--- @param cursor string|nil -- cursorId of the last item in the previous page
+--- @param limit number
+--- @param sortBy string|nil
+--- @param sortOrder string|nil
+--- @return PaginatedTable
+function gar.getPaginatedVaultsFromAllGateways(cursor, limit, sortBy, sortOrder)
+	--- @type DelegatesFromAllGateways[]
```

Comment:
Confusing type choice since this is for gateway vaults rather than delegate vaults?

Reply:
ah good catch, some copy paste here

Reply:
dff38b7